m2m/m2m.py

```diff
@@ -185,62 +181,52 @@ def writeJSONFile(self, baseDirectory, foldername, data):
 
 if __name__ == '__main__':
 
-    usage = 'usage: %prog [options] <csv file>'
+    parser = ArgumentParser()
 
-    parser = OptionParser(usage=usage)
+    parser.add_argument('mapping',
+                        help='Specify the mapping file to use for this CSV')
 
-    parser.add_option('-m', '--mapping', action='store', type='string',
-                      dest='mapping',
-                      help='Specify the mapping file to use for this CSV')
+    parser.add_argument('file',
+                        help='Specify a CSV file to process.')
 
-    parser.add_option('-n', '--row', action='store', type='int',
-                      dest='row',
-                      help='Specify a single row number to process')
+    parser.add_argument('-n', '--row', type=int,
+                        dest='row',
+                        help='Specify a single row number to process')
 
-    parser.add_option('-w', '--write', action='store_true',
-                      dest='write',
-                      help='Write records to files')
+    parser.add_argument('-w', '--write', action='store_true',
+                        dest='write',
+                        help='Write records to files')
 
-    parser.add_option('-j', '--json',
-                      action='store_true',
-                      dest='json',
-                      help='UWrite json version of metadata')
+    parser.add_argument('-j', '--json', action='store_true',
+                        dest='json',
+                        help='Write json version of metadata')
 
```

Comment:
Personally, I think the blank lines in this section are a bit much to have between all of these `parser` lines and should be used more [sparingly](https://www.python.org/dev/peps/pep-0008/#blank-lines) as these fit together into a logical section and have heavy indenting to separate them visually already.